orchestration/dags/import_analytics.py

```diff
@@ -69,6 +70,49 @@
     "criterion",
 ]
 
+date_to_convert = [
+    [
+        "user_creation_date",
+        "user_cultural_survey_filled_date",
+        "user_last_connection_date",
+    ],
+    "",
+    ["offerer_modified_at_last_provider_date", "offerer_creation_date"],
+    "dateModified",
+    ["booking_creation_date", "booking_used_date", "booking_cancellation_date"],
+    "",
+    ["venue_modified_at_last_provider", "venue_creation_date"],
+    "",
+    ["offer_modified_at_last_provider_date", "offer_creation_date"],
+    [
+        "stock_modified_at_last_provider_date",
+        "stock_modified_date",
+        "stock_booking_limit_date",
+        "stock_creation_date",
+    ],
+    "dateCreated",
+    "",
+    "",
+    "date",
+    "",
+    "date",
+    "date",
+    "dateCreated",
+    "",
+    "dateCreated",
+    "",
+    "",
+    "dateModifiedAtLastProvider",
+    "",
+    "",
+    "",
+    "",
+    "",
+    "",
+]
+
+tables_to_convert = dict(zip(data_applicative_tables, date_to_convert))
```

Comment:
je pense qu'il est plus simple et plus rigoureux de créer directement le dictionnaire ayant pour clefs le nom des tables, et en valeurs la liste des colonnes de dates à convertir.

Reply:
En faisant comme ça j'avais en tête de perturber au minimum le fonction des autres task qui itère sur 'data_applicative_tables'